docs/applications/emby.md

```diff
@@ -0,0 +1,107 @@
+install for Ansible-NAS provides a server, which various clients can then
+connect to from various platforms - other computers, smartphones, smart TVs,
+etc.
+
+Note that [Plex](https://www.plex.tv/), also included in Ansible-NAS, has a very
+similar functionality.
+
+## Usage
+
+Set `emby_enabled: true` in your `group_vars/all.yml` file. There are further
+parameters you can edit such as `movies_root` and `tv_root` lower down in this
+file. 
+
+## Specific Configuration
+
+The emby web interface can be found at port 8096 (http) or 8920 (https, if
+configured) of your NAS. After installation, point your browser there to start
+the configuration wizard. For instance, on a NAS with the IPv4 address
+192.168.1.2, browse to `http://192.168.1.2:8096`. Heimdall has a dedicated icon
+for emby. 
+
+By default, Ansible-NAS gives emby read/write access to the folders where your
+movies and TV shows are stored. To change this to read-only, edit the following
+lines in `tasks/emby.yml`
+
+```
+      - "{{ emby_movies_directory }}:/movies:rw"
+      - "{{ emby_tv_directory}}:/tv:rw"
+```
+
+so that they end in `ro` instead of `rw`. Note that emby will not be able to
+delete files then, which might be exactly what you want. However, you will not
+have the option to store cover art in the related folders. Always leave the
+configuration directory read/write. 
+
+## File system considerations
+
+You might want to create special ZFS datasets for TV shows and movies, as these
+are usually large files. This means you can change the record size parameter
+from its default value of 128K to 1M. Larger blocks cut down on the metadata
+that the system has to manage, as various parts of ZFS work on the block level. 
+
+Assuming we have a ZFS pool named `tank`, the instruction sequence might look
+something like this:
+
+```
+        sudo zfs create tank/movies
+        sudo zfs set recordsize=1M tank/movies
+
+        sudo zfs create tank/tv
+        sudo zfs set recordsize=1M tank/tv
+```
+
+Changing the record size will not affect files already stored in the data set,
+only new files. See [this
+discussion](https://blog.programster.org/zfs-record-size) about record sizes for
+media data sets and [this
+video](https://www.youtube.com/watch?v=SJB1cJfcjYI&feature=youtu.be&t=14m41s)
+for more information. 
+
+Since video files are almost always compressed, you might want to turn off
+`compression` for the data set. However, ZFS automatically detects data that
+can't be compressed and aborts the procedure. 
+
+> At time of writing, native encryption of ZFS datasets is not supported in the
+> release versions of ZFS on Linux (ZoL), but is scheduled for [version
+> 0.8.0](https://github.com/zfsonlinux/zfs/releases/tag/zfs-0.8.0-rc3) sometime
+> in 2019. It is unclear when Ubuntu will upgrade their version of ZFS.
+
+To see all properties of a ZFS dataset, use `zfs get all <DATASET>`.
+
+## Naming movies and TV shows
```

Comment:
There must be a better way of doing this than telling people to rename all video files?

Reply:
(Laughs hysterically) You'd think so in the age of AI, wouldn't you? Sadly, from my experience with Plex, later moved to Emby, no -- you can either name the files the way the system wants it or you can manually match them later through the interface.

The problem, to be fair, is that too much stuff has the same name. Take "The Avengers", which means Scarlett Johansson in leather for some people and Diana Rigg in leather for others. Then of course there is https://www.imdb.com/title/tt0034639/?ref_=fn_tt_tt_7 from 1942, though I'm not sure if anybody is wearing leather. I stopped counting on IMDB at 20 films with the name "Dracula". You get the picture.